Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,141 @@
+                print("\\     /")
+                print("  '---'")
+                print("  _|_|_")
+                condition = false
+            }
+            
+        } else {
+            
+            wrongLetterCounter += 1
+            
+            print("Wrong guess")
+            print(String(dashes))
+            print(wrongLetterCounter)
+            
+            if wrongLetterCounter == 1 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("Strike 1. You have 5 guesses left.")
+            } else if wrongLetterCounter == 2 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("|        |")
+                print("Strike 2. You have 4 guesses left.")
+            } else if wrongLetterCounter == 3 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("|        |\\")
+                print("Strike 3. You have 3 guesses left.")
+            }
+            if wrongLetterCounter == 4 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("|       /|\\")
+                print("Strike 4. You have 2 guesses left.")
+            }
+            if wrongLetterCounter == 5 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("|       /|\\")
+                print("|         \\")
+                print("Strike 5. You have 1 FINAL!!! guess left.")
+            }
+            if wrongLetterCounter == 6 {
+                print("__________")
+                print("|        |")
+                print("|        0")
+                print("|       /|\\")
+                print("|       / \\")
+            }
+        }
+        
+        if wrongLetterCounter == 6 {
+            print("You lost better luck next time")
+            condition = false
+            print(randomWord)
+            
+        }
+        //      print("condition \(condition)")
+        
+    } while condition
+    
+    
+    repeat {
+        print("Want to play again? Type Yes or No")
+        
+        let answer = readLine()?.lowercased() ?? ""
+        
+        switch answer {
+        case "yes":
+            print("yes")
+            reapeatGame = true
+            makeAChoice = false
+            wrongLetterCounter = 0
+            randomWord = allTheWords.randomElement() ?? ""
+            dashes = Array(repeating: "_" ,count: randomWord.count)
+            
+            continue startloop
+        case "no":
+            print("Goodbye")
+            reapeatGame = false
+            makeAChoice = false
+           // break
+        default:
+            print("Enter Yes or No, please!")
+        }
+        
+    } while makeAChoice
+    
```

Comment:
Review your nested loops and when they are kicking in, your while makeAChoice is kicking in immediately after your while repeatGame is going into effect.